salt/modules/pacmanpkg.py

```diff
@@ -186,7 +179,20 @@ def version(*names, **kwargs):
         salt '*' pkg.version <package name>
         salt '*' pkg.version <package1> <package2> <package3> ...
     """
-    return __salt__["pkg_resource.version"](*names, **kwargs)
+    foo = __salt__["pkg_resource.version"](*names, **kwargs)
```

Comment:
was this on purpose? Asking, because `foo` is usually used for demonstration purposes.